message_parser/src/main/java/cumulus_message_adapter/message_parser/JsonUtilities.java

```diff
@@ -0,0 +1,37 @@
+package cumulus_message_adapter.message_parser;
+
+import java.util.Map;
+
+import java.lang.reflect.Type;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+/**
+ * Utilities for converting JSON to string/map
+ */
+public class JsonUtilities {
+
+  /**
+   * Convert json string to Map object
+   *
+   * @param jsonString The json string
+   * @return The converted Map object
+   */
+  public static Map<String,Object> convertJsonStringToMap(String jsonString) {
+      Gson gson = new Gson();
+      Type mapObjectType = new TypeToken<Map<String, Object>>() {}.getType();
+      return gson.fromJson(jsonString, mapObjectType);
```

Comment:
No need for declaring `gson`. Just use inline instantiation: `return new Gson().fromJson(...)`